infra/config-store/src/test/java/com/simprints/infra/config/store/local/models/FingerprintConfigurationTest.kt

```diff
@@ -58,18 +58,6 @@ class FingerprintConfigurationTest {
             assertThat(it.value.toProto()).isEqualTo(it.key)
         }
     }
-    @Test
```

Comment:
But how about the coverage :D

Reply:
The allowed age groups was already covered in other tests